test/input_gen/genInput.py

```diff
@@ -577,8 +578,11 @@ def gen_test_case_bn(input_shape, base_name, training=True):
                 base_name = "tc_fc_1")
 
 # Bn layer unit test case:
-    if target == "bn_1":
-        gen_test_case_bn(input_shape = [3, 1, 4, 5], base_name = "tc_bn_1")
+    if target == "bn_fc_1":
+        gen_test_case_bn(input_shape = [3, 1, 1, 12], base_name = "tc_bn_fc_1", axis=-1)
+
+    if target == "bn_conv_2":
+        gen_test_case_bn(input_shape = [3, 2, 4, 5], base_name = "tc_bn_conv_2", axis=1)
 
```

Comment:
make a unittest with batch size 1 as well.

Reply:
I'll do this in other PR as even this tc is not used as of now.

Reply:
ok.